OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -47,7 +47,7 @@ function Get-TargetResource {
         [string]$TaskLogsDirectory = "$HomeDirectory\TaskLogs",
         [bool]$LogTaskMetrics = $false,
         [bool]$LogRequestMetrics = $false,
-        [int]$TaskCap = $null
```

Comment:
i was trying to support a “not configured” state with the `$null`, so that if it wasn’t set, it implied that it we don't want the DSC to manage it.